app/routes/index.js

```diff
@@ -284,4 +310,71 @@ module.exports = function(app, fs) {
 			else
 				res.redirect('/admin');
 		});
+	app.post('/adduser', upload.single('profile_pic'), async function(req,res){
+		try {
+			const user = new User(req.body);
+			console.log(user)
+			if(req.file){
+				user.profile_pic = req.file.tempFilePath;
+			}
+			await user.save()
+			res.json("User added successfully");
+		} catch (error) {
+			console.log("error");
+			res.json(error);
+			
+		}
+		
+	});
+	app.route('/addevent').
+	post(adminAuth, async function(req,res){
+		try {
+			const event = new Event(req.body);
+			event.save()
+			res.json(event);
+		} catch (error) {
+			console.log("error");
+			res.json(error);
+			
+		}
+	})
+	app.route(adminAuth, '/addteamhere').
```

Comment:
Can you please rename this to just '/team'

Reply:
Done